app/models/generic_event/move_notify_premises_of_arrival_in_30_mins.rb

```diff
@@ -1,5 +1,9 @@
 class GenericEvent
   class MoveNotifyPremisesOfArrivalIn30Mins < GenericEvent
     eventable_types 'Move'
+
+    def event_classification
```

Comment:
Another option would be to add an interim ancestor class e.g. `GenericMoveNotifyEvent` and add this code there, making the three existing move notify classes then inherit from that. There's a similar approach to this already used for `Incident` so fits quite nicely with the existing OO style and means less repeated code. Up to you as it's mostly a style preference though...